kategory-optics/src/test/kotlin/kategory/optics/instances/NonEmptyListInstancesTest.kt

```diff
@@ -0,0 +1,30 @@
+package kategory.optics
+
+import io.kotlintest.KTestJUnitRunner
+import io.kotlintest.properties.Gen
+import kategory.Eq
+import kategory.LensLaws
+import kategory.Option
+import kategory.UnitSpec
+import kategory.genFunctionAToB
+import kategory.genNonEmptyList
+import org.junit.runner.RunWith
+
+@RunWith(KTestJUnitRunner::class)
+class NonEmptyListInstancesTest : UnitSpec() {
+
+    init {
+
```

Comment:
Could you please add a test for instance lookup? Check the ones in IdTest for example.

Reply:
These "instances" are no typeclasses so no lookup occurs.